src/features/DeFiZap/components/ZapCard.tsx

```diff
@@ -215,9 +216,22 @@ interface Props {
   config: IZapConfig;
 }
 
+// Moved from StoreProvider
+const getDeFiAssetReserveAssets = (poolAsset: StoreAsset, assets: StoreAsset[]) => (
+  getPoolAssetReserveRate: (poolTokenUuid: string, assets: Asset[]) => ReserveAsset[]
+) =>
+  getPoolAssetReserveRate(poolAsset.uuid, assets).map((reserveAsset) => ({
+    ...reserveAsset,
+    balance: multiplyBNFloats(
+      weiToFloat(poolAsset.balance, poolAsset.decimal).toString(),
+      reserveAsset.reserveExchangeRate
+    ),
+    mtime: Date.now()
+  }));
+
```

Comment:
tested ?

Reply:
It's a copy paste from the `StoreProvider` for a deprecated feature.
Request to skip test requirements.

Reply:
I'm okay with it :)